src/js/view/view.js

```diff
@@ -1027,6 +1027,10 @@ define([
             _controlbar.setAltText(text);
         };
 
+        this.showControls = function() {
+            utils.removeClass(_playerElement, 'jw-flag-ads-hide-controls');
+        };
```

Comment:
Now you can use `this.showControls()` in `this.destroyInstream` in place of `utils.removeClass(_playerElement, 'jw-flag-ads-hide-controls');`

or don't call utils.removeClass twice - it accepts arrays and space delimited strings:

`utils.removeClass(_playerElement, ['jw-flag-ads', 'jw-flag-ads-hide-controls']);`